octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/preferences/PluginPreferenceStorage.java

```diff
@@ -56,7 +56,7 @@ public interface PrefereceChangeHandler {
     }
 
     public interface PreferenceConstants {
-        public static final String OCTANE_SERVER_URL = "OpenText™️ Core Software Delivery PlatformServerUrl";
+        public static final String OCTANE_SERVER_URL = "octaneServerUrl";

```

Comment:
huh? was it bad before?

Reply:
it was _octaneServerUrl_ long ago - modified to be back to it